.travis.yml

```diff
@@ -121,9 +121,7 @@ matrix:
         - python: 2.7
           env: DOWNSTREAM=aws-encryption-sdk
         - python: 2.7
-          env: DOWNSTREAM=dynamodb-encryption-sdk OPENSSL=1.1.0j
-          # dynamodb-encryption-sdk tests fail on xenial for whatever reason
-          dist: trusty
+          env: DOWNSTREAM=dynamodb-encryption-sdk OPENSSL=1.1.0j BOTO_CONFIG=/dev/null
           sudo: false
```

Comment:
You can remove this line now too

Reply:
Good call. I left it in because https://github.com/boto/boto/issues/3717 seemed to indicate that it was also needed, but testing on my fork confirms it's fine without it.